docs/site-example/site.conf

```diff
@@ -209,8 +209,13 @@
     -- },
     -- define if the contact field is obligatory (optional)
     -- owner = {
-      -- obligatory = true
+      -- obligatory = true,
     -- },
+    -- Show/hide the password login
+    -- remote_login = {
```

Comment:
Did we really want to add all possible values to the `site-example`?

In my opinion there is no need for package dependent configuration in the `site-example`, when there is already an example in `docs/packages`

Reply:
Only examples that are interesting should be listed in the site-example. I added this, because many communities will wonder where the password login went and want it back.